node/core/provisioner/src/lib.rs

```diff
@@ -150,12 +150,7 @@ pub enum Error {
 
 /// Provisioner run arguments.
 #[derive(Debug, Clone, Copy)]
-pub struct ProvisionerConfig {
-	/// If enabled, dispute votes will be provided to `fn create_inherent`, otherwise not.
-	/// Long term we will obviously always want disputes to be enabled, this option exists for testing purposes
-	/// and will be removed in the near future.
-	pub disputes_enabled: bool,
-}
+pub struct ProvisionerConfig;
```

Comment:
Should we get rid of this? looks dangling.

Reply:
I'd argue keeping it, it's more expressive than `()` and has zero cost.

Reply:
That makes sense if we expect to add some configuration options there in the future, else I'd prefer to remove stuff that ain't gonna be used.

Reply:
I actually had a version where I removed it and switched to `()` but as @drahnr points out, it's not expressive. Unit structs are pretty standard in Rust